pallets/offchain-worker/src/tests.rs

```diff
@@ -1,315 +1,271 @@
-//		"15EW3AMRm2yP6LEF5YKKLYwvphy3DmMqN6":{"final_balance":1220,"n_tx":4,"total_received":310925609}
-//	}"#;
-//	assert_eq!(Some(vec![1220]), urls::parse_blockchain_info_balances(double_balances));
-//
-//	// Test case should fail because first balance value is negative
-//	let double_balances = r#"
-//	{
-//		"1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa":{"final_balance":-30,"n_tx":2635,"total_received":6835384571},
-//		"15EW3AMRm2yP6LEF5YKKLYwvphy3DmMqN6":{"final_balance":1220,"n_tx":4,"total_received":310925609}
-//	}"#;
-//	assert_eq!(Some(vec![1220]), urls::parse_blockchain_info_balances(double_balances));
-//}
-//
-//#[test]
-//fn test_parse_infura_balances() {
-//	let double_balances = r#"
-//	[
-//		{"jsonrpc":"2.0","id":1,"result":"0x4563918244f40000"},
-//		{"jsonrpc":"2.0","id":1,"result":"0xff"}
-//	]
-//	"#;
-//
-//	assert_eq!(Some(vec![5000000000000000000, 255]), urls::parse_infura_balances(double_balances));
-//}
+// This file is part of Substrate.
+
+// Copyright (C) 2020-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
 //
-//#[test]
-//fn test_parse_infura_balances_2() {
-//	let double_balances = r#"
-//	[
-//		{"jsonrpc":"2.0","id":1,"result":"0x4563918244f40000"},
-//		{"jsonrpc":"2.0","id":1,"result":"0xff"}
-//	]
-//	"#;
-//	let token_info: Vec<urls::InfuraBalance> = serde_json::from_str(double_balances).unwrap();
-//	assert_eq!(token_info[0].id, 1);
+// 	http://www.apache.org/licenses/LICENSE-2.0
 //
-//}
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::*;
+use crate as pallet_offchain_worker;
+use frame_support::parameter_types;
+use sp_core::{ H256, sr25519::Signature,};
+use sp_runtime::{
+	testing::{Header, TestXt},
+	traits::{
+		BlakeTwo256, IdentityLookup, Extrinsic as ExtrinsicT,
+		IdentifyAccount, Verify,
+	},
+};
 
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+// For testing the module, we construct a mock runtime.
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		System: frame_system::{Module, Call, Config, Storage, Event<T>},
+		Balances: pallet_balances::{Module, Call, Storage, Config<T>, Event<T>},
+		AccountLinker: account_linker::{Module, Call, Storage, Event<T>},
+
+		Example: pallet_offchain_worker::{Module, Call, Storage, Event<T>,},
```

Comment:
Example may need to be renamed in the next change.
Also account_linker and pallet_offchain_worker may have a unified naming convention, either with or without pallet_ prefix.

Reply:
agree, will fix it in next pr, attribute macro for account linker.